cypress/integration/ui_spec.js

```diff
@@ -1172,4 +1172,53 @@ describe("UI", () => {
 			});
 		});
 	});
+	describe("localstorage", () => {
+		it("should only have device and account registration", () => {
+			const parleyConfig = {userAdditionalInformation: {"some-key": "some-value"}};
+			const testMessage = `test message localstorage variable check ${Date.now()}`;
+			visitHome(parleyConfig);
+
+			cy.get("[id=app]").as("app");
+
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			const newAuthHeader = "1234";
+			const apiVersion = "v1.6";
+
+			// see configuration options https://developers.parley.nu/docs/settings#runoptions
+			cy.window().then((win) => {
+				// eslint-disable-next-line no-param-reassign
+				win.parleySettings.authHeader = newAuthHeader;
+				// eslint-disable-next-line no-param-reassign
+				win.parleySettings.apiVersion = apiVersion;
+			});
```

Comment:
Ik wil een aparte test voor ingelogd.
Dus een test die een bericht stuurt zonder in te loggen en dan checken wat er in de localstorage zit
en daarnaast een test die een bericht stuurt waarbij je ingelogd bent en checked dat de authHeader er niet in zit
(btw beter om de auth header te zetten vóórdat je de chat start)

Reply:
Ik heb van de test twee aparte testen gemaakt en de parleySettings eerst doorgevoerd alsvorens de chat wordt gestart d18d6431915bf6e880d64815d341b2140d51bc80